great-idea/css/index.css

```diff
@@ -62,3 +62,220 @@ h1, h2, h3, h4, h5 {
     letter-spacing: 1px;
     margin-bottom: 15px;
 }
+
+h1 {
+	font-size: 80px;
+}
+
+.container {
+	width: 880px;
+	margin: 0 auto;
```

Comment:
One way to make your file even more like the mockup would be to add a margin to the top and sides of the container element.